docs/resources/oxide_silo.md

```diff
@@ -0,0 +1,83 @@
+---
+page_title: "oxide_silo Resource - terraform-provider-oxide"
+---
+
+# oxide_silo (Resource)
+
+This resource manages the creation of an Oxide Silo.
```

Comment:
Let's update the documentation to note that `quotas` is the only attribute that's able to be updated.